staging/publishing/rules.yaml

```diff
@@ -1646,3 +1646,18 @@ rules:
       dir: staging/src/k8s.io/controller-manager
     name: release-1.19
     go: 1.14.6
+
+- destination: component-helpers
+  library: true
+  branches:
+  - source:
+      branch: master
+      dir: staging/src/k8s.io/component-helpers
+    name: master
+    dependencies:
+    - repository: apimachinery
+      branch: master
+    - repository: api
+      branch: master
+    - repository: client-go
```

Comment:
client-go would need to be removed from here because component-helper doesn't import it (and client-go isn't in it's go.mod)

Reply:
thanks!!!